<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="51696" started-at="2018-03-10T19:35:16Z" finished-at="2018-03-10T19:36:08Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="51696" started-at="2018-03-10T19:35:16Z" finished-at="2018-03-10T19:36:08Z">
      <class name="com.practice.hybrid.sessionsuite.ChangePasswordTest">
        <test-method status="FAIL" signature="changePasswordTest(java.util.Hashtable)[pri:0, instance:com.practice.hybrid.sessionsuite.ChangePasswordTest@3cd3e762]" name="changePasswordTest" duration-ms="50783" started-at="2018-03-11T01:05:16Z" data-provider="getData" finished-at="2018-03-11T01:06:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{NewPassword=Temp@123, OldPassword=Test@123, ExpectedResult=Success, Runmode=Y, Browser=Mozilla}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
	at org.junit.Assert.fail(Assert.java:93)
	at com.practice.hybrid.Keywords.executeKeywords(Keywords.java:96)
	at com.practice.hybrid.sessionsuite.ChangePasswordTest.changePasswordTest(ChangePasswordTest.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- changePasswordTest -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.practice.hybrid.sessionsuite.ChangePasswordTest@3cd3e762]" name="quit" is-config="true" duration-ms="793" started-at="2018-03-11T01:06:07Z" finished-at="2018-03-11T01:06:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.practice.hybrid.sessionsuite.ChangePasswordTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
